src/Api/API.csproj

```diff
@@ -1,5 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
That's inside `Directory.Build.props`, so it should already have the same effect?